plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -1946,7 +1953,7 @@ SecRule REQUEST_FILENAME "@rx /apps/deck/cards/[0-9]+(?:/reorder|/unassign)?$" \
 
 # Deleting stacks, boards, labels, and cards
 # Deleting a label from a card
-SecRule REQUEST_FILENAME "@rx /apps/deck/(?:boards|cards|cards/\d+/label|labels|stacks)/\d+$" \
+SecRule REQUEST_FILENAME "@rx /apps/deck/(?:boards|cards|cards/[0-9]+/label|labels|stacks)/[0-9]+$" \
```

Comment:
This is fine of course, but why did you make this change?

Reply:
It's easier to read, I kept on mixing up the forward and back slashes when writing the regex.